Steepfile

```diff
@@ -4,10 +4,6 @@ target :lib do
   signature "sig"
 
   check "app"
-
-  configure_code_diagnostics do |hash|
-    hash[D::Ruby::UnknownInstanceVariable] = :hint
-  end
```

Comment:
With this change, we can disable suppressing the `UnknownInstanceVariable` diagnostic.